### DISCUSSION TITLE
Disable IPv6 in Netbox Image

### DISCUSSION THREAD

**Discussion by aalsayegh** [netbox-community/netbox-docker#1234] [Getting Help]
I have IPv6 disabled on my server. Netbox-Docker enables IPv6 in Netbox image by default. So I'm getting the following error message when executing docker compose up:

Applying configuration from /etc/unit/nginx-unit.json
[alert] 7#7 socket(\"[::]:8080\") failed (97: Address family not supported by protocol)
[alert] 95#95 failed to apply new conf
Could no load Unit configuration
dependency failed to start: container netbox-docker-netbox-1 exited (0)

I added the following to docker-compose.override.yml but didn't fix:

networks:
  nginx:
    driver:  bridge
    enable_ipv6: false

Any suggestion to resolve?

**Comment by bandi22** [comment]
I am experiencing the very same issue.

Even [enabling](https://docs.docker.com/engine/daemon/ipv6/) the ipv6 support of the Docker daemon didn't help.

**Comment by bandi22** [comment]
This solution helped:
https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting#netbox-container-doesnt-start-when-ipv6-is-disabled
